types/textfit/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "lib": ["ES5", "DOM"],
+        "module": "commonjs",
+
+        "noEmit": true,
+        "types": [],
+
+        "baseUrl": "../",
+        "typeRoots": ["../"],
+
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+
+        "esModuleInterop": true,
```

Comment:
Please remove or set esModuleInterop to false to simulate using this package with a legacy project that doesn't have it turned on.